filter/store.go

```diff
@@ -41,11 +43,28 @@ type HTTPClient interface {
 }
 
 // PutCSVData allows the filtered file data to be sent back to the filter store when complete.
-func (store *Store) PutCSVData(filterJobID string, url string, size int64) error {
+func (store *Store) PutCSVData(filterJobID string, fileURL string, size int64) error {
 
-	// todo: make call to the filter API.
+	url := store.filterAPIURL + "/filters/" + filterJobID
+
+	putBody := &observation.Filter{
+		State: "completed",
+		Downloads: &observation.Downloads{
+			CSV: &observation.DownloadItem{
+				Size: strconv.FormatInt(size, 10),
```

Comment:
The size field is an int64. The typical 'strconv.itoa' function takes only the int type, not int64